grid2op/multi_agent/multiAgentEnv.py

```diff
@@ -198,8 +200,15 @@ def __init__(self,
         
         
     def reset(self) -> MADict:
-        # TODO
-        raise NotImplementedError()
+        # TODO : done, tested
+        self._cumulative_rewards = dict(
+            zip(
+                self.agents, [0. for _ in range(self.num_agents)]
+            )
+        )
+        self._cent_observation = self._cent_env.reset()
+        self._update_observations()
+        return self.observations
     
     def _handle_illegal_action(self, reason):
```

Comment:
this and _handle_ambiguous_action might not be totally correct.

In particular, as you did not think about "what to do" and "how to do it" you have been forced to add something like "ambiguous_except_tmp" in the key of a dict sent to the agent. 

Not the best to have a "temporary variable" as output of a function. And when I say "not the best" I mean "terrible practice"

Reply:
Removed it. We'll treat it future versions

Reply:
I removed ambiguous_except_tmp